### PR TITLE
AA - Add District view

### PR DESCRIPTION
**Pull request by echaidemenos**
## How to test the feature:

- [ ]
- [ ]

## TODO
- select on map click

## Checklist - did you ...

Test your changes with

- [ ] `REACT_APP_COUNTRY=rbd yarn start`
- [ ] `REACT_APP_COUNTRY=cambodia yarn start`
- [ ] `REACT_APP_COUNTRY=mozambique yarn start`
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

## Screenshot/video of feature:
Screenshot from 2024-03-20 17-07-15

### CODE REVIEW

**Comment by echaidemenos** [comment]
@ericboucher a couple comments:
- what do these actions are and do
image
- same about this filter 'emergency triggers' 
image

Also not that the data from this old CSV look a bit random. Shouldn't the severity only be increasing as time goes by?

File: frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/DistrictView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  }, [dispatch]);
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.districtViewWrapper}>
+        {windows.map(x => (
+          <WindowColumn key={x} win={x} selectedDistrict={selectedDistrict} />
+        ))}
+      </div>
+      <div className={classes.actionsWrapper}>
```
Comment (ericboucher) [inline frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/DistrictView/index.tsx:186]: The actions boxes should be aligned with the months I think

**Comment by ericboucher** [comment]
@echaidemenos the icons for the actions can be found here
https://www.figma.com/file/Xe8H1GuVMJUsIi4JNTYPVo/WFP-Anticipatory-Action-(AA)?type=design&node-id=15-226&mode=design&t=UkZUVCuzfBJHcJx6-0
It is a mix of Ocha (Font Awesome) and MUI icons


And here is the list of actions:

W means window

Ready alert of moderate and severe within w1 and w2 = launching tenders and other Procurement procedures

Icon set Moderate W1 = Signing contracts and mobilization of assets; Dissemination of early warning messages; Distribution of seeds for re-seeding; Rehabilitation of water holes; Transformation of water holes into multi-use systems

Icon Set Moderate W2 = Signing contracts and mobilization of assets; Dissemination of early warning messages; Distribution of seeds and vegetative material for the second season; Vaccination of birds against New Castle; Social assistance

Icon Set Severe W1 = Signing contracts and mobilization of assets; Dissemination of early warning messages; Moving Cattle to places with water availability; Rehabilitation of water holes; Transformation of water holes into multi-use systems

Icon Set Severe W2 = Signing contracts and mobilization of assets; Dissemination of early warning messages; Food Assistance; School snack; Social assistance

File: frontend/src/config/types.ts [hunk 2]
```diff
@@ -851,6 +851,7 @@ export type UserAuth = {
 };
 
 export enum PanelSize {
+  undef = '',
```
Comment (ericboucher) [inline frontend/src/config/types.ts:854]: what does that translate to?

**Pull request merged**